app/exchanges/volume_tracking/message_builder.rb

```diff
@@ -33,7 +33,7 @@ def publish_data # rubocop:disable Metrics/MethodLength
 
       case aliquot.used_by_type
       when 'Pacbio::Well'
-        data[:used_by_type] = 'well'
+        data[:used_by_type] = 'run'
         data[:used_by_barcode] =
           "#{aliquot.used_by.plate.sequencing_kit_box_barcode}:#{aliquot.used_by.plate.plate_number}:#{aliquot.used_by.position}" # rubocop:disable Layout/LineLength
```

Comment:
There seems to be a few of these. You can put the line over 2 lines using "..." \ which means you won't have to disable.